src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\DnsMock;
+
+require_once __DIR__.'/../DnsMock.php';
+
+class DnsMockTest extends \PHPUnit_Framework_TestCase
+{
+    protected function tearDown()
+    {
+        DnsMock::withMockedHosts(array());
+    }
+
+    public function testCheckdnsrr()
+    {
+        DnsMock::withMockedHosts(array('example.com' => array(array('type' => 'MX'))));
```

Comment:
Understood, what about the test for the `get_dns_record()` function then? Of the entries one would expect to retrieve (namely ``host`, `type`, `class`, and `ttl`) we only have the `type` entry here. The `class` value, for example, is always the same and could be added by our mock implcitly if it isn't present.